axelor-project/src/main/java/com/axelor/apps/project/service/ProjectServiceImpl.java

```diff
@@ -145,4 +157,58 @@ public BigDecimal computeDurationFromChildren(Long projectId) {
     q.setParameter("projectId", projectId);
     return q.getSingleResult();
   }
+
+  @Override
+  @Transactional
+  public void createProjectFromTemplate(Project project, ProjectTemplate projectTemplate) {
```

Comment:
remove project as parameter and add clientPartner and code,  method can be call with null value for those two